docs/content/Caching/Getting-Started-Pre-Aggregations.md

```diff
@@ -159,7 +159,175 @@ the pre-aggregations are rebuilt.
 These refreshes can be done on-demand, or [in the background as a scheduled
 process][ref-caching-preaggs-bk-refresh].
 
+## Ensuring pre-aggregations are targeted by queries
+
+### Additivity
+
+So far, we've described pre-aggregations as aggregated versions of your existing
+data.
+
+However, there are some rules that apply to when Cube.js uses the
+pre-aggregation. The "additivity" of fields specified in both the query and in
+the pre-aggregation determine this.
+
+So what is additivity? Let's add another cube called `LineItems` to the previous
+example to demonstrate. The `LineItems` "belong to" the `Orders` cube, and are
+[joined][ref-schema-joins] as such:
+
+```javascript
+cube(`LineItems`, {
+  sql: `SELECT * FROM public.line_items`,
+
+  joins: {
+    Orders: {
+      sql: `${CUBE}.order_id = ${Orders}.id`,
+      relationship: `belongsTo`,
+    },
+  },
+
+  measures: {
+    count: {
+      type: `count`,
+      drillMembers: [id, createdAt],
+    },
+  },
+
+  dimensions: {
+    id: {
+      sql: `id`,
+      type: `number`,
+      primaryKey: true,
+    },
+
+    createdAt: {
+      sql: `created_at`,
+      type: `time`,
+    },
+  },
+});
+```
+
+Some sample data from the `line_items` table might look like:
+
+| **id** | **product_id** | **order_id** | **quantity** | **price** | **profit_margin** | **created_at**             |
```

Comment:
I love ❤️ the way you crafted this example and explanation. Just gorgeous!

Reply:
Can't take the credit for that - @keydunov came up with the idea 😄